apprise/plugins/NotifyEmail.py

```diff
@@ -540,6 +545,10 @@ def __init__(self, smtp_host=None, from_name=None,
                 '({}) specified.'.format(recipient),
             )
 
+        if not reply_to:
+            # Add ourselves to the Reply-To directive
+            self.reply_to.add(self.from_addr)
```

Comment:
I don't understand why set Reply-To with from addr, it's the default behaviour without Reply-To.
We need to set Reply-To when we want to reply to another address than the sender.

Reply:
Ah, i was trying to just make it set the flag in all circumstances.  I thought it would be consistent. I didn't realize it got placed anyway.  We can roll this back; no big deal.

Reply:
Reply-To header is not placed by default, but it must be using only when reply address sould be different than sender.